odoo/addons/base/models/res_currency.py

```diff
@@ -97,6 +97,15 @@ def _deactivate_group_multi_currency(self):
         if group_user and group_mc:
             group_user.sudo()._remove_group(group_mc.sudo())
 
+    @api.constrains('active')
+    def _check_company_currency_stays_active(self):
+        if self._context.get('install_mode') or self._context.get('force_deactivate'):
```

Comment:
why do you check install_mode in the context? :thinking: 

Also, I think this line deserves a comment to explain why we would like not the check the constraint

Reply:
Comment added. install_mode is checked because at install, when this check is run, the "active" field of a currency added to a company will still be evaluated as False, despite it's automatically set at True when added to the company.